backend/danswer/background/celery/tasks/indexing/tasks.py

```diff
@@ -482,6 +484,15 @@ def connector_indexing_task(
     that the task transitioned to a "READY" state but the generator_complete_key doesn't exist.
     This will cause the primary worker to abort the indexing attempt and clean up.
     """
+    if SENTRY_DSN:
```

Comment:
nit: can we add a comment as to why this is needed here but not for other celery-based code sections. I figured it out but it took me a few seconds to realize